### PR TITLE
fixed user functional tests to resolve AB#14798

### PR DESCRIPTION
**Pull request by ytqsl**
# Fixes or Implements [AB#14798](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14798)
[Task 14978](https://dev.azure.com/qslvic/Health%20Gateway/_workitems/edit/14978): Fix Cypress Functional Tests - ui/user and ui/timeline

## Description

Fixed functional tests

## Testing

- [ ] Unit Tests Updated
- [x] Functional Tests Updated
- [ ] Not Required

## UI Changes

None

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Testing/functional/tests/cypress/integration/ui/user/dependents.js [hunk 1]
```diff
@@ -16,7 +16,22 @@ describe("COVID-19", () => {
...
-        cy.enableModules(["CovidLabResults", "Laboratory", "Dependent"]);
+        cy.configureSettings({
+            dependents: {
+                enabled: true,
+            },
+            datasets: [
+                {
+                    name: "laboratory",
+                    enabled: true,
+                },
```
Comment (MikeLyttle) [inline Testing/functional/tests/cypress/integration/ui/user/dependents.js:27]: This dataset can be removed.  It's now "covid19TestResult" which is already enabled here.

File: Testing/functional/tests/cypress/integration/ui/user/dependents.js [hunk 2]
```diff
@@ -114,12 +129,26 @@ describe("COVID-19 - Vaccine Proof download", () => {
...
+
+        cy.configureSettings({
+            dependents: {
+                enabled: true,
+            },
+            datasets: [
+                {
+                    name: "laboratory",
+                    enabled: true,
+                },
```
Comment (MikeLyttle) [inline Testing/functional/tests/cypress/integration/ui/user/dependents.js:141]: This dataset can be removed.  It's now "covid19TestResult" which is already enabled here.

File: Testing/functional/tests/cypress/integration/ui/user/dependents.js [hunk 3]
```diff
@@ -167,12 +196,26 @@ describe("COVID-19 - Vaccine Proof download", () => {
...
```
Comment (MikeLyttle) [inline Testing/functional/tests/cypress/integration/ui/user/dependents.js:208]: This dataset can be removed.  It's now "covid19TestResult" which is already enabled here.

**Review by BrianMaki — APPROVED** [review]

File: Testing/functional/tests/cypress/integration/ui/user/dependents.js [hunk 4]
```diff
@@ -202,11 +245,18 @@ describe("Dependents - Immunization Tab - Enabled", () => {
...
+            dependents: {
+                enabled: true,
+            },
+            datasets: [
+                {
+                    name: "immunization",
+                    enabled: true,
+                },
+            ],
+        });
```
Comment (MikeLyttle) [inline Testing/functional/tests/cypress/integration/ui/user/dependents.js:258]: The "immunization" dataset also needs to be enabled for dependents.

File: Testing/functional/tests/cypress/integration/ui/user/dependents.js [hunk 5]
```diff
@@ -441,11 +491,26 @@ describe("Dependents - Lab Results Tab - Enabled", () => {
...
+            },
+            datasets: [
+                {
+                    name: "laboratory",
+                    enabled: true,
+                },
+                {
+                    name: "covid19TestResult",
+                    enabled: true,
+                },
```
Comment (MikeLyttle) [inline Testing/functional/tests/cypress/integration/ui/user/dependents.js:506]: These datasets can be removed.

File: Testing/functional/tests/cypress/integration/ui/user/dependents.js [hunk 6]
```diff
@@ -441,11 +491,26 @@ describe("Dependents - Lab Results Tab - Enabled", () => {
...
+                {
+                    name: "labResult",
+                    enabled: true,
+                },
```
Comment (MikeLyttle) [inline Testing/functional/tests/cypress/integration/ui/user/dependents.js:510]: The "labResult" dataset also needs to be enabled for dependents.

File: Testing/functional/tests/cypress/integration/ui/user/dependents.js [hunk 7]
```diff
@@ -510,11 +575,18 @@ describe("Dependents - Clinical Document Tab - Enabled", () => {
...
-        ]);
+        cy.configureSettings({
+            dependents: {
+                enabled: true,
+            },
+            datasets: [
+                {
+                    name: "clinicalDocument",
+                    enabled: true,
+                },
```
Comment (MikeLyttle) [inline Testing/functional/tests/cypress/integration/ui/user/dependents.js:586]: The "clinicalDocument" dataset also needs to be enabled for dependents.

File: Testing/functional/tests/cypress/integration/ui/user/profile.js [hunk 8]
```diff
@@ -4,7 +4,6 @@ const HDID = "P6FFO433A5WPMVTGM7T4ZVWBKCSVNAYGTWTU3J2LWMGUMERKI72A";
 
 describe("User Profile", () => {
     beforeEach(() => {
-        cy.enableModules([]);
```
Comment (MikeLyttle) [inline Testing/functional/tests/cypress/integration/ui/user/profile.js:7]: You might want to replace this with `cy.configureSettings({});` just to ensure everything unimportant is disabled.

**Review by MikeLyttle — CHANGES_REQUESTED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Pull request merged**